Latest release does not compile on Windows (on AppVeyor at least).

The CI build of Fava is currently broken due to the latest release of Beancount apparently not compiling on Windows (https://ci.appveyor.com/project/yagebu/fava/builds/34396347):

    beancount/parser/parser.c(336): error C2198: 'PyUnicode_FromString': too few arguments for call

Installing from git (branch v2) seems to fix it (https://ci.appveyor.com/project/yagebu/fava/builds/34397338). The line that throws the error is gated by an `#ifdef VC_CHANGESET` and maybe the passing of that compile option behaves differently on Windows.

--- a/beancount/parser/macros.h
+++ b/beancount/parser/macros.h
@@ -1,6 +1,12 @@
 #ifndef BEANCOUNT_PARSER_MACROS_H
 #define BEANCOUNT_PARSER_MACROS_H
 
+#define STRINGIFY(x) XSTRINGIFY(x)
+/* Prepend the empty string to work around MSVC bug for which an empty
+ * define results in no string rather than in the empty string. */
+#define XSTRINGIFY(x) "" #x
+
+
 #define _CC_FUNC_01(FN, X, ...) FN(X)
 #define _CC_FUNC_02(FN, X, ...) FN(X); _CC_FUNC_01(FN, __VA_ARGS__)
 #define _CC_FUNC_03(FN, X, ...) FN(X); _CC_FUNC_02(FN, __VA_ARGS__)

--- a/beancount/parser/parser.c
+++ b/beancount/parser/parser.c
@@ -4,6 +4,7 @@
 #define PY_SSIZE_T_CLEAN
 #include <Python.h>
 
+#include "macros.h"
 #include "parser.h"
 #include "grammar.h"
 #include "lexer.h"
@@ -323,17 +324,17 @@ PyMODINIT_FUNC PyInit__parser(void)
 
     /* Hash of the this Python extension source code. */
     SETATTR(module, "SOURCE_HASH",
-            PyUnicode_FromString(Py_STRINGIFY(PARSER_SOURCE_HASH)));
+            PyUnicode_FromString(STRINGIFY(PARSER_SOURCE_HASH)));
 
     /* Release versions as defined in setup.py. */
     SETATTR(module, "__version__",
-            PyUnicode_FromString(Py_STRINGIFY(RELEASE_VERSION)));
+            PyUnicode_FromString(STRINGIFY(RELEASE_VERSION)));
 
 #ifdef VC_CHANGESET
     /* Git changeset from the build source tree.
      * In the Bazel build, this information is absent. */
     SETATTR(module, "__vc_changeset__",
-            PyUnicode_FromString(Py_STRINGIFY(VC_CHANGESET)));
+            PyUnicode_FromString(STRINGIFY(VC_CHANGESET)));
 #endif
 
 #ifdef VC_TIMESTAMP

